dmpr/dmpr.py

```diff
@@ -15,6 +15,12 @@
 
 logger = logging.getLogger(__name__)
 
+mylog = logging.getLogger("mylog")
+fh = logging.FileHandler('/var/logs/tmp/myLog.txt')
```

Comment:
the path is completly b0rken:

- please let the user the ability where to log, not silently log somewhere. You should log to stderr if not known
- the directory is probably not available
- /var/logS is not a valid path /var/log (but see other comment)
- please always name the logfile of path to your application, nobody known that myLog is dmpr
- no camel case for linux filenames, that is not linux (see FHS pdf, the linux standard)

Reply:
the logger is unnecessary anyways (just for debug purposes) - deleted for the merge